source/locates.html.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Locates FIX API Specification
+
+
+toc_footers:
+  - [Powered By Slate](https://github.com/slatedocs/slate)
+
+search: true
+---
+
+# Introduction
+Locates FIX API specification details the tags and values along with description of values for each field for processing locate requests and acceptances from OMS clients. Clear Street currently utilizes FIX 4.2 format.
+
+# Configuring FIX Session
+```
+// Example Config
+[DEFAULT]
+ReconnectInterval=5
+SenderCompID=LOCATESCLIENT
+HeartBtInt=30
+SocketAcceptPort=31234
+TargetCompID=CLST
+HeartBtInt=30
+SocketConnectPort=31234
+SocketConnectHost=100.0.0.0
+BeginString=FIX.4.2
+
+[SESSION]
+StartTime=04:00:00
+EndTime=22:00:00
+```
+A FIX session is defined as a unique combination of a `BeginString` (the FIX version number 4.2), a `SenderCompID` (OMS Client ID as defined by Clear Street), and a `TargetCompID`.
```

Comment:
What does Comp mean here in 'TargetCompID' and 'SenderCompID'

Reply:
These two are the terms used in FIX. Target Company ID and Sender Company ID.